runtime/v2/manager.go

```diff
@@ -356,8 +356,55 @@ func (m *MM[T]) TxValidators() func(ctx context.Context, tx T) error {
 	}
 }
 
-// TODO write as descriptive godoc as module manager v1.
-// TODO include feedback from https://github.com/cosmos/cosmos-sdk/issues/15120
+// RunMigrations performs in-place store migrations for all modules. This
```

Comment:
This has simply copied/pasted from types/module, but this is not totally accurate anymore.

Reply:
updated